pdf/pdfcanvas.cpp

```diff
@@ -113,9 +113,16 @@ PDFCanvas::PDFCanvas(QQuickItem* parent)
 
 PDFCanvas::~PDFCanvas()
 {
-    if(d->resizeTimer)
-        delete d->resizeTimer;
+    for( int i = 0; i < d->pageCount; ++i )
```

Comment:
Is that ```d->pageCount``` really needed. Would the pages.count() be enough (with a temporary variable). Just increases bookkeeping.